doc/MAGIC.md

```diff
@@ -476,6 +476,28 @@ You can assign a spell as a special attack for a monster.
 
 ```
 
+To add the enchantment to the item, you need to write the id of your enchantment inside `relic_data` field. For example:
+
+```json
+"relic_data": { "passive_effects": [ { "id": "ench_fishform" } ] },
+
+```
+
+If your enchantment is relatively small, you can write it right in the thing json, using the same syntaxys as the common enchantment. For example:
+
+```json
+"relic_data": {
+  "passive_effects": [
+    {
+      "has": "WORN",
+      "condition": "ALWAYS",
+      "values": [ { "value": "ARMOR_BULLET", "add": 8 }, { "value": "ARMOR_STAB", "add": 3 }, { "value": "ARMOR_CUT", "add": 3 } ]
```

Comment:
I thought I'd corrected that item in a recent Magiclysm fix but all other ARMOR enchantments are in negative numbers. 

```
"passive_effects": [
        {
          "has": "WORN",
          "condition": "ALWAYS",
          "values": [
            { "value": "ARMOR_CUT", "add": -4 },
            { "value": "ARMOR_BASH", "add": -4 },
            { "value": "ARMOR_STAB", "add": -4 },
            { "value": "ARMOR_BULLET", "add": -2 }
          ]
          ```

Reply:
Hmm, i guess i picked the numbers from my master fork, which is a bit outdated